core/shared/settings-cache/public.js

```diff
@@ -29,5 +29,13 @@ module.exports = {
     members_support_address: 'members_support_address',
     members_enabled: 'members_enabled',
     members_invite_only: 'members_invite_only',
-    paid_members_enabled: 'paid_members_enabled'
+    paid_members_enabled: 'paid_members_enabled',
+    firstpromoter: 'firstpromoter',
+    firstpromoter_id: 'firstpromoter_id',
```

Comment:
Do we really need both a bool and an ID? Can we just have one?

Reply:
We can do with just `firstpromoter_id`, but we'll need to add a tiny logic in front somewhere that sets it to `null` when `firstpromoter` is turned off, as right now both are set independently, so its possible to have `firstpromoter` set to `false` but still an old id to present atm. 😅

Reply:
Oh I could probably just update this in the new Calculated fields that we have now 💡

Reply:
Updated with new CalculatedSetting `firstpromoter_account` that is `null` if integration is off or the value otherwise. ✅